src/components/DetailsPage/DetailsPage.js

```diff
@@ -0,0 +1,103 @@
+import { MdNavigateBefore } from 'react-icons/md'
+import { Link } from 'react-router-dom'
+import styled from 'styled-components/macro'
+import CustomLink from '../CustomLink/CustomLink'
+
+function DetailsPage({
+  name,
+  image,
+  brand,
+  description,
+  ingredients,
+  nutritionFacts,
+  stores,
+  categories,
+  price,
+  id,
+}) {
+  return (
+    <>
+      <Link to={`/product-details/${id}`} />
+      <Heading>{name}</Heading>
```

Comment:
I am not sure, but did you want to have the heading within a link? Because right now, there is a link, but it doesn't surround anything, so nothing can really happen